tests/test_cli.py

```diff
@@ -678,6 +678,35 @@ def test_run_non_universal_newline(self):
         self.assertEqual(
             (ctx.returncode, ctx.stdout, ctx.stderr), (1, expected_out, ''))
 
+    def test_run_list_files(self):
+        with RunContext(self) as ctx:
+            cli.run(('--list-files', self.wd))
+        self.assertEqual(ctx.returncode, 0)
+        self.assertEqual(''.join(sorted(ctx.stdout.splitlines(True))), (
```

Comment:
We are sorting the output to sidestep sorting issues of the filesystem/Python. We are doing something similar when testing `find_files_recursively`.

I opted to keep newlines and use `''.join` to ensure that every line has a new line at the end.

Reply:
> We are sorting the output to sidestep sorting issues of the filesystem/Python. We are doing something similar when testing `find_files_recursively`.

:+1: 

> I opted to keep newlines and use `''.join` to ensure that every line has a new line at the end.

I don't think this is needed (we don't check this for any other yamllint output), and it makes the test harder to read. Would it be OK for you to keep same format as:
https://github.com/adrienverge/yamllint/blob/6194a28/tests/test_cli.py#L153-L158
?

Reply:
@adrienverge Sure, let's do this :+1: